mmdet3d/models/losses/monoflex_iou_loss.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch import nn as nn
+
+from mmdet.models.builder import LOSSES
+from mmdet.models.losses.utils import weighted_loss
+
+
+@weighted_loss
+def monoflex_iou_loss(pred, target, mode='iou', eps=1e-7):
```

Comment:
Can refactor to re-use IoU loss from mmdet

Reply:
I will remove the IoU loss in this PR.